pkg/collector/container_hc_collector.go

```diff
@@ -34,6 +34,11 @@ import (
 // #define CPU_VECTOR_SIZE 128
 import "C"
 
+var (
+	ebpfBatchGet            = true
+	ebpfBatchGetSize uint32 = 1000
```

Comment:
didn't see this honored somewhere, but seems we should handle this e.g 2500 records so we get 1000,1000,500?

Reply:
right, this is not used. will fix